demisto_sdk/commands/validate/validators/PB_validators/PB115_is_tasks_quiet_mode.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsTasksQuietModeValidator(BaseValidator[ContentTypes]):
+    error_code = "PB115"
+    description = "Checks if the 'quietmode' field of all tasks in playbook are not in default value."
+    rationale = "Confirmation of turning off quitmode"
+    error_message = "Playbook '{playbook_name}' contains tasks that are not in quiet mode (quietmode: 2) The tasks names is: '{tasks}'."
+    fix_message = (
+        "Fixed playbook '{playbook_name}' to set tasks '{tasks}' to (quietmode: 0)."
+    )
+    related_field = "tasks"
+    is_auto_fixable = True
+    related_file_type = [RelatedFileType.YML]
+    invalid_tasks_in_playbooks = {}
+
+    def get_invalid_task_ids(self, content_item: ContentTypes) -> List:
+        """
+        Identify tasks with quietmode == 2 and update self.invalid_tasks_in_playbooks with these tasks.
+
+        Args:
+            content_item (ContentTypes): The content item to check.
+
+        Returns:
+            List[str]: List of task IDs where quietmode == 2.
+        """
+        invalid_task_ids = [
+            task
+            for task in content_item.tasks.values()
+            if task.quietmode == 2
+        ]
+        if invalid_task_ids:
+            self.invalid_tasks_in_playbooks[content_item.name] = invalid_task_ids
```

Comment:
```suggestion
        self.invalid_tasks_in_playbooks[content_item.name] = invalid_task_ids
```
We don't mind writing it in any case, worst case we'll write an empty list into it.